### ISSUE TITLE
Improper GUI error on JDK version compatibility

### ISSUE THREAD

**Issue by Aravinth-Earth** [zdila/gpx-animator#411]
## Update:
The below listed issue resolved(By updating my local openJDK version to 18) - But a minor case needs clarity - see the last comment (Is this an valid/expected error in this situation?)
### System Information:
- OS: Ubunut 22.04 LTS
- Java openjdk 11.0.15 2022-04-19 | OpenJDK Runtime Environment (build 11.0.15+10-Ubuntu-0ubuntu0.22.04.1) | OpenJDK 64-Bit Server VM (build 11.0.15+10-Ubuntu-0ubuntu0.22.04.1, mixed mode, sharing)

### Issue Details:
1. When tried render with gpx file added in GUI, throws error `could not get tile https://www.bing.com/maps/`
    - But when checked the site is accessible directly in browser
    - Also in GUI, Form File Menu, On clicking on Download Map Updates button  it says `Successfully completed > Data from 1099 maps have been downloaded and are now available in GPX Animator`
    - - So no clue on what is the issue here!
1. When tried with CLI way, throws 
```
Error: LinkageError occurred while loading main class app.gpx_animator.Main
	java.lang.UnsupportedClassVersionError: app/gpx_animator/Main has been compiled by a more recent version of the Java Runtime (class file version 60.0), this version of the Java Runtime only recognizes class file versions up to 55.0
```

### Next Steps:
- Planning to build from source, meanwhile trying to see if someone has any solutions for the current state

**Comment by Aravinth-Earth** [comment]
It was actually an JDK version compatibility issue in my local machine, when installed with OenJDK 18 using `sudo apt install openjdk-18-jdk` the issue got resolved - NO issue in GUI or CLI way

**Comment by Aravinth-Earth** [comment]
But still, initially the GUI error seems not thrown properly!

When there in a java compatibility issue, the GUI says `could not get tile https://www.bing.com/maps/`. 

Is this an valid/expected error in this situation?

**Comment by McPringle** [comment]
Bing Maps are actually not supported because Microsoft decided to use a proprietary protocol. GPX Animator accidentally imported too many maps including maps with unsupported protocols. This is fixed in the master branch already and will be published with the upcoming v1.8. For Bing support, there is a feature request open for implementing the proprietary protocol: #391 

GPX Animator v1.7 needs Java 16+, GPX Animator v1.8 will need Java 17+.
